FindRange.java

```diff
@@ -1,16 +1,48 @@
-/*
- * File: FindRange.java
- * Name: 
- * Section Leader: 
- * --------------------
- * This file is the starter file for the FindRange problem.
- */
-
-import acm.program.*;
-
-public class FindRange extends ConsoleProgram {
-	public void run() {
-		/* You fill this in */
-	}
-}
-
+/*
+ * File: FindRange.java
+ * Name: 
+ * Section Leader: 
+ * --------------------
+ * This file is the starter file for the FindRange problem.
+ */
+
+
+
+		import acm.program.*;
+
+		public class FindRange extends ConsoleProgram {
+			private static final int Sentinel = 0;

```

Comment:
the convention for naming static variables is all caps. So SENTINEL instead of Sentinel